lib/rules/font-weight-notation/README.md

```diff
@@ -82,3 +82,25 @@ The following patterns are *not* considered violations:
 a { font-weight: 400; }
 a b { font-weight: lighter; }
 ```
+
+### `ignoreFunctions: ["/regex/", "string"]`
+
+Given:
+
+```js
+["weight", "/^my-/", "/^YOUR-/i"]
+```
+
+The following patterns are not considered violations:
+
+```css
```

Comment:
Please break this into separate code fences, one per each example e.g.

```css
strong {
  font-weight: weight('light');
}
```

```css
strong {
  font-weight: my-weight('light');
}
```

That's the normal convention. (The other example in this README needs updating, but not in this PR)